disperser/dataapi/server.go

```diff
@@ -402,6 +418,49 @@ func (s *server) FetchOperatorsNonsigningPercentageHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, metric)
 }
 
+// FetchDeregisteredOperators godoc
+//
+//	@Summary	Fetch list of DeregisteredOperators for days
+//	@Tags		OperatorsInfo
+//	@Produce	json
+//	@Success	200		{object}	BlobsResponse
+//	@Failure	400		{object}	ErrorResponse	"error: Bad request"
+//	@Failure	404		{object}	ErrorResponse	"error: Not found"
+//	@Failure	500		{object}	ErrorResponse	"error: Server error"
+//	@Router		/operatorsInfo/deRegisteredState [get]
+func (s *server) FetchDeregisteredOperators(c *gin.Context) {
+	timer := prometheus.NewTimer(prometheus.ObserverFunc(func(f float64) {
+		s.metrics.ObserveLatency("FetchDeregisteredOperators", f*1000) // make milliseconds
+	}))
+	defer timer.ObserveDuration()
+
+	// Get query parameters
+	// Default Value 14 days
+	days := c.DefaultQuery("days", "14") // If not specified, defaults to 14
```

Comment:
do we want to validate this to note be greater than like 30 days? since we can put 10000 here and query might take too long? just an idea